### PR TITLE
Centos-8 support in Hubble 3.0

### PR DESCRIPTION
**Pull request by goravsingal**
Centos-8 support in Hubble 3.0

### CODE REVIEW

File: pkg/centos8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+#fpm package making requirements start
+RUN dnf install -y ruby ruby-devel rpm-build rubygems gcc make \
+ && gem install --no-ri --no-rdoc fpm
+
+#pyinstaller start
+#commands specified for ENTRYPOINT and CMD are executed when the container is run, not when the image is built
+#use the following variables to choose the version of hubble
+ENV HUBBLE_CHECKOUT=v3.0.11
+ENV HUBBLE_VERSION=3.0.11
```
Comment (devagupt) [inline pkg/centos8/Dockerfile]: please confirm if it will be 3.0.11 or 3.0.12

Comment (goravsingal) [reply pkg/centos8/Dockerfile]: Done, please check

File: pkg/centos8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+       --iteration ${HUBBLE_ITERATION} \
+       --url ${HUBBLE_URL} \
+       --description \"${HUBBLE_DESCRIPTION}\" \
+       --rpm-summary \"${HUBBLE_SUMMARY}\" \
+       --after-install /hubble_build/conf/afterinstall-systemd.sh \
+       --after-upgrade /hubble_build/conf/afterupgrade-systemd.sh \
+       --before-remove /hubble_build/conf/beforeremove.sh \
+       etc/hubble opt usr /var/log/hubble_osquery/backuplogs \
+#edit to change iteration number, if necessary
+    && cp hubblestack-${HUBBLE_VERSION}-${HUBBLE_ITERATION}.x86_64.rpm /data/hubblestack-${HUBBLE_VERSION}-${HUBBLE_ITERATION}.el7.x86_64.rpm \
```
Comment (praksinha) [inline pkg/centos8/Dockerfile]: @goravsingal extension should be .el8.x86_64.rpm

File: pkg/centos8/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+    && openssl dgst -sha256 /data/hubblestack-${HUBBLE_VERSION}-${HUBBLE_ITERATION}.el7.x86_64.rpm \
```
Comment (praksinha) [inline pkg/centos8/Dockerfile]: Same here, please change extension to **.el8**.x86_64.rpm

File: pkg/centos8/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+                          > /data/hubblestack-${HUBBLE_VERSION}-${HUBBLE_ITERATION}.el7.x86_64.rpm.sha256" ]
```
Comment (praksinha) [inline pkg/centos8/Dockerfile]: same here

File: pkg/centos8/Dockerfile (hunk 2 above)
Comment (goravsingal) [reply pkg/centos8/Dockerfile]: done

File: pkg/centos8/Dockerfile (hunk 4 above)
Comment (goravsingal) [reply pkg/centos8/Dockerfile]: done

File: pkg/centos8/Dockerfile (hunk 3 above)
Comment (goravsingal) [reply pkg/centos8/Dockerfile]: done

Comment (praksinha) [reply pkg/centos8/Dockerfile]: thanks

**Pull request merged**